build.sh

```diff
@@ -2,6 +2,12 @@
 
 set -e
 
+if [ "$(uname)" == "Darwin" ]; then
+    #Mac OS X platform
```

Comment:
>Mac OS X platform [](start = 5, length = 17)

Very minor note, I  *think* they've deprecated the name "Mac OS X" in favor of "macOS".